packages/cursorless-vscode/src/keyboard/KeyboardConfig.ts

```diff
@@ -91,23 +95,43 @@ export class KeyboardConfig {
   >(
     tokenType: T,
     sectionName: K = tokenType as unknown as K,
-    only?: V[],
+    filter?: (value: V) => boolean,
```

Comment:
this `V` was the wrong type. It should have been `T`, because that's the input type